src/app/admin/components/filter-search/filter-search.component.scss

```diff
@@ -0,0 +1,317 @@
+      }
+    }
+
+    &.waiting {
+      color: #5ec9da;
+
+      .indicator {
+        background: #5ec9da;
+      }
+
+      &:hover {
+        color: white;
+        background: #5ec9da;
+
+        .indicator {
+          background: white;
+        }
+      }
+    }
+
+    &.review {
+      color: #f5a623;
+
+      .indicator {
+        background: #f5a623;
+      }
+
+      &:hover {
+        color: white;
+        background: #f5a623;
+
+        .indicator {
+          background: white;
+        }
+      }
+    }
+
+    &.released {
+      color: $green;
+
+      .indicator {
+        background: $green;
+      }
+
+      &:hover {
+        color: white;
+        background: $green;
+
+        .indicator {
+          background: white;
+        }
+      }
+    }
+
+    &.rejected {
+      color: saturate($error-red, 15);
+
+      .indicator {
+        background: saturate($error-red, 15);
+      }
+
+      &:hover {
+        color: white;
+        background: saturate($error-red, 15);
+
+        .indicator {
+          background: white;
+        }
+      }
+    }
+  }
+}
+
+.filter-status__container {
+  li {
+    position: relative;
+    color: $dark-text;
+
+    .indicator {
+      width: 6px;
+      height: 6px;
+      border-radius: 50%;
+      display: inline-block;
+      background: $dark-text;
+      position: absolute;
+      left: 10px;
+      top: 0;
+      bottom: 0;
+      margin: auto;
+    }
+
+    .svg-inline--fa,
+    .icon {
+      margin-left: 15px;
+    }
+
+    &:hover {
+      color: white;
+      background: $dark-text;
+
+      .indicator {
+        background: white;
+      }
+    }
+
+    &.waiting {
+      color: #5ec9da;
+
+      .indicator {
+        background: #5ec9da;
+      }
+
+      &:hover {
+        color: white;
+        background: #5ec9da;
+
+        .indicator {
+          background: white;
+        }
+      }
+    }
+
+    &.proofing {
+      color: #bd5eda;
+    
+
+    .indicator {
+        background: #bd5eda;
+      }
+
+    &:hover {
+      color: white;
+      background: #bd5eda;
+
+      .indicator {
+          background: white;
+        }
+      }
+    }
+
+    &.review {
+      color: #f5a623;
+
+      .indicator {
+        background: #f5a623;
+      }
+
+      &:hover {
+        color: white;
+        background: #f5a623;
+
+        .indicator {
+          background: white;
+        }
+      }
+    }
+
+    &.released {
+      color: $green;
+
+      .indicator {
+        background: $green;
+      }
+
+      &:hover {
+        color: white;
+        background: $green;
+
+        .indicator {
+          background: white;
+        }
+      }
+    }
+
+    &.rejected {
+      color: saturate($error-red, 15);
+
+      .indicator {
+        background: saturate($error-red, 15);
+      }
+
+      &:hover {
+        color: white;
+        background: saturate($error-red, 15);
+
+        .indicator {
+          background: white;
+        }
+      }
+    }
+  }
+}
+
+.loading-template {
```

Comment:
Why is this loading template here? To my knowledge the filters menu doesn't loading anything asynchronously?